docs/pages/api-docs/timeline-item-opposite-content.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import MarkdownDocs from 'docs/src/modules/components/MarkdownDocs';
+import { prepareMarkdown } from 'docs/src/modules/utils/parseMarkdown';
+
+const pageFilename = 'api/timeline-item-opposite-content';
+const requireRaw = require.context('!raw-loader!./', false, /\/timeline-item-opposite-content\.md$/);
```

Comment:
It's the first time I see this instability between `yarn prettier` and `yarn docs:api`.
Maybe it should tell us that the name of the component is too long? What about we turn
`<TimelineItemOppositeContent />` into `<TimelineItemContent position="opposite" />`?

Reply:
Renaming the component to `TimelineOppositeContent` solved the issue